armi/nuclearDataIO/rtflux.py

```diff
@@ -61,14 +61,19 @@
 
 class RtfluxData:
     """
-    Data structure that is read from or written to a RTFLUX file.
+    Multigroup flux as a function of i,j,k and g indices.
+
+    The metadata also contains the power and k-eff.
+
+    This is the data structure that is read from or written to a RTFLUX file.
     """
 
     def __init__(self):
         # Need Metadata subclass for default keys
         self.metadata = nuclearFileMetadata._Metadata()
 
-        self.groupFluxes = None
+        self.groupFluxes: numpy.ndarray = numpy.array([])
+        """Maps i,j,k,g indices to total real or adjoint flux in n/cm^2-s"""
```

Comment:
do these indices correspond to our grid indexing?

Reply:
in some rare cases, maybe. But almost certainly not in most cases. for example in Finite difference cases there will be 6 or more of these per axial submesh of each block, and so on.

We could reconsider our grid indices based on these if we want since these have a longer legacy and there were smarter people doing nuclear models back then

Reply:
Probably worth mentioning whatever class could be used to make sense of the i, j, k indices